libs/ai-endpoints/langchain_nvidia_ai_endpoints/chat_models.py

```diff
@@ -456,11 +490,12 @@ def _custom_postprocess(
             "response_metadata": {},
         }
         if token_usage := kw_left.pop("token_usage", None):
-            out_dict["usage_metadata"] = {
-                "input_tokens": token_usage.get("prompt_tokens", 0),
-                "output_tokens": token_usage.get("completion_tokens", 0),
-                "total_tokens": token_usage.get("total_tokens", 0),
-            }
+            if (streaming and stream_usage) or not streaming:
```

Comment:
how about always returning tokens usage if its in the response?

Reply:
that makes sense, because does not matter parameter at this point